src/datasources/helpers/paginationQuery.ts

```diff
@@ -0,0 +1,42 @@
+import { sql } from "drizzle-orm";
+import { PgSelect } from "drizzle-orm/pg-core";
+
+import { ORM_TYPE } from "~/datasources/db";
+
+export const paginationDBHelper = async <
+  T extends { execute: (...args: any) => any },
+>(
+  DB: ORM_TYPE,
+  select: T,
+  {
+    page,
```

Comment:
should we provide defaults for these? e.g `page` = 0, `pageSize` = 30

Reply:
🔥